fixtures/docker/docker-kafka-generate-ssl-certs/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8u322-jdk-buster
```

Comment:
Is it possible to use an alpine-slim image to speed up the build or is there a reason not to?

Reply:
Not sure if there is an alpine-slim image for JDK 8. Will have a look.

I need to use JDK 8, because the Kafka container uses that version underneath and certificates generated in newer versions don't seem to be supported.

Reply:
Switched to using `openjdk:8-slim` image.